File: TAREA1.cs
using System;

class Program {
    static void Main() {
        int[] arr = {40, 50, 60, 70, 80, 90};
        Console.Write("Recorrido lineal (secuencial): ");
        Console.Write("\nLos elementos del array son: ");
        foreach (var x in arr) {
            Console.Write(x + " ");
        }
        Console.WriteLine();
    }
}

File: TAREA1.cpp
#include <iostream>
using namespace std;

int main() {
    int arr[] = {40, 50, 60, 70, 80, 90};
    cout << "Recorrido lineal (secuencial): ";
    cout << "\nLos elementos del array son: ";
    for (int x : arr) {
        cout << x << ' ';
    }
    cout << '\n';
    return 0;
}